src/components/Tank/Tank.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from 'react';
+import { getBgPosition, getTankPositions } from './Tank.helpers';
+import { TankProps } from './Tank.model';
+import styles from './Tank.module.scss';
+
+const propsDefault: Readonly<Required<TankProps>> = {
+  color: 'yellow',
+  position: 'up',
+  type: '0'
+};
+
+// eslint-disable-next-line complexity
+const Tank: FC<TankProps> = (
+  props: Readonly<TankProps>
+): JSX.Element => {
+  const { xPos, yPos } = getTankPositions(
+    props.color ?? propsDefault.color,
+    props.position ?? propsDefault.position,
+    props.type ?? propsDefault.type
+  );
+
+  const spriteStyle: React.CSSProperties = {
```

Comment:
- [ ] I don't feel comfortable with doing stuff like this in React. Did you try doing that in the CSS?